### PR TITLE
Genoanime presence added

### PR DESCRIPTION
**Pull request by Genoanime**
Code Analytics added

### CODE REVIEW

File: websites/G/GenoAnime/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "id": "424072110647017482"
+  },
+  "url": "genoanime.com",
+  "description": {
+    "en": "Enjoy the our website, view any anime episode and movie"
+  },
+  "service": "GenoAnime",
+  "version": "1.0.0",
+  "logo": "https://genoanime.com/premid/geno.jpg",
+  "thumbnail": "https://genoanime.com/premid/thumb.png",
```
Comment (Bas950) [inline websites/G/GenoAnime/dist/metadata.json:14]: upload to imgur

File: websites/G/GenoAnime/presence.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+var presence = new Presence({
+  clientId: "810203651317432351"
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+var title: any;
+var titleName: any;
+var search: any;
```
Comment (Bas950) [inline websites/G/GenoAnime/presence.ts:9]: no var no any

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/GenoAnime/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "DayAndNight",
+    "id": "424072110647017482"
+  },
+  "url": "genoanime.com",
+  "description": {
+    "en": "Enjoy the our website, view any anime episode and movie"
+  },
+  "service": "GenoAnime",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/bkcZfjM.jpeg",
```
Comment (Bas950) [inline websites/G/GenoAnime/dist/metadata.json:13]: logo must be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working,
and add some more things to the presence, right now this wouldn't get merged since its a low use/quality presence

File: websites/G/GenoAnime/dist/metadata.json (hunk 3 above)
Comment (Genoanime) [reply websites/G/GenoAnime/dist/metadata.json]: image

image

**Comment by Genoanime** [comment]
Uploaded proof of it working

**Comment by ririxidev** [comment]
low quality presence.

**Pull request closed without merging**